packages/sudoku-utils/src/utils/border-clues.ts

```diff
@@ -69,3 +92,77 @@ export function getBorderCluesToDraw(clue: Borderclue): Borderclue[] {
     }
   ];
 }
+
+/*Checks the inputted digits against the standard constraint logic for the border clue
+and returns any cells that have errors*/
+export function verifyBorderClue(borderclue: Borderclue, solution: CellValues): Position[] {
+  let isValid = true;
+
+  if (!(borderclue.nonStandard ?? false)) {
```

Comment:
Do an early return here instead
```suggestion
  if ((borderclue.nonStandard ?? false)) {
    return []
  }
  
```

Reply:
done